src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts

```diff
@@ -52,7 +52,10 @@ describe('ContainerSelectorComponent', () => {
                     identifier: '427c47a4-c380-439f',
```

Comment:
migrate to `TestBed`

Reply:
done https://github.com/dotCMS/core-web/pull/1419/commits/9f0329227f49d5e2ba679ec450bf7170112f69e3#diff-f80340d948cb93edaa053f4b5ac39829R38